### ISSUE TITLE
Breadcrumbs missing aria-label="Breadcrumbs"

### ISSUE THREAD

**Issue by rajanjangratelus** [telus/tds-core#1704]
## Description

- Using VoiceOver on Mac OS, Breadcrumbs components reads out as "Link" only. There is no label to identify it as a Breadcrumb. 
- VoiceOver identify Breadcrumbs <link name> Link

## Reproduction Steps

1. Turn on VoiceOver
2. Go to [TELUS.com
](https://www.telus.com/en/business/small/mobility?linktype=ge-meganav)
3. Tab to Breadcrumbs component. VoiceOver will read out Links.

## Recommendation

- Adding the aria-label="Breadcrumbs" would make VoiceOver say: Breadcrumbs <link name> Link

## Meta

- TDS component version: 
- Willing to develop solution: Yes
- High impact: Yes/No 

## Screenshots

example: Apple's website has their breadcrumb labelled.

<img width="1585" alt="Screen Shot 2022-01-20 at 12 38 02 PM" src="https://user-images.githubusercontent.com/84352103/151185029-a9d3ea7f-28c6-4784-8fa3-93315bdbaa41.png">
<img width="717" alt="Screen Shot 2022-01-20 at 12 53 42 PM" src="https://user-images.githubusercontent.com/84352103/151185035-5ffa0f85-3ed9-481a-9d52-5ad9f7e2433b.png">

**Comment by billtelus** [comment]
I'll look into it...
Did you check if it's not fixed already in `UDS` ?

**Comment by owestinTelus** [comment]
Hopefully helpful context for this and the Allium/UDS component
[WAI-ARIA Authoring Practice 1.2: Breadcrumb example](https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html) 
> The separators are part of the visual presentation that signifies the breadcrumb trail, which is already semantically represented by the nav element with its label of Breadcrumb. So, using a display technique that is not represented in the accessibility tree used by screen readers prevents redundant and potentially distracting verbosity.

**Comment by billtelus** [comment]
This issue have been resolved and will now be closed
